views/live.ejs

```diff
@@ -0,0 +1,236 @@
+			}
+			.Blink {
+				animation: blinker 1.5s cubic-bezier(.5, 0, 1, 1) infinite alternate;  
+			}
+			@keyframes blinker {  
+				from { opacity: 1; }
+				to { opacity: 0; }
+			}
+
+	</style>
+	<a href="/event/view/<%=eventId%>" ><button style="margin-top:10px;" class="btn btn-primary">Event Page</button> </a>
+	<div class="milestone" id ="milestonePopup"></div>
+	<div id="milestoneText"></div>
+	<body class="container">
+		<div class="row" style="margin-top:20px">				
+				<div class="col-md-6" style="text-align:center;">					
+					<div class="">
+						<div><i class="fa fa-circle text-danger Blink"></i>&nbsp; LIVE </div>
+						<h2 style="word-break: break-all; " > <%= result.title %> , Live Updates!</h2>			
+					</div>			
+					<br>
+					<div class="form-group">
+						<h3>Milestones</h3>
+						<%if(isAdminUser){%>
+						<button type="button" class="btn btn-info" data-toggle="modal" data-target="#addModal">Add Milestone</button>							
+						<%}%>
+						<% var existsGoal = true; if(!milestones||milestones.amounts.length==0){ existsGoal =false; %>
+							<div>-No milestones-</div>
+						<%}else{%>
+						<%for(var x=0;x<milestones.amounts.length;x++){%>
+							<div style="font-size:1.5em">
+								€<span class="milestones" data-ach="<%=milestones.amounts[x].achieved%>"><%=milestones.amounts[x].amount%></span>								
+								<span><%=milestones.amounts[x].label%></span>
+								<span><% if(milestones.amounts[x].achieved){%>
+									<span class='achieved'>Achieved</span>
+								<% }else{ %>
+									<span class='notachieved'>Not reached yet</span>
+								<%}%>
+									</span>
+							</div>
+						<%}}%>
+						<%if(isAdminUser){%>
+						
+						<div class="row">
+							<form action="/live/add/<%=eventId%>/<%=existsGoal%>" method="post" id='form'>
+								<div id="addModal" class="modal fade" role="dialog">
+									<div class="modal-dialog">
+										<div class="modal-content">
+											<div class="modal-header">
+												<button type="button"style="float:left;margin:0;" class="close" data-dismiss="modal">&times;</button>
+												<h4 class="modal-title" style="float:left">Add Milestone</h4>
+											</div>
+											<div class="modal-body" >
+												<label>Milestone Label</label>
+												<input type="label" name="label" class="form-control" value="" rows="5" required><br>
+												<label>Donation goal</label>
+												<input type="number" step="0.01" min="1" max = "9999999" name="amount" class="form-control" value="" rows="5" required><br>							
+											</div>
+											<div class="modal-footer">		
+												<button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>							
+												<button type="submit" class="btn btn-success">Confirm</button>
+											</div>
+										</div>
+									</div>
+								</div>		
+							</form>	
+						</div>
+						<%}%>
+
+					</div>
+					<div class="form-group">
+						<h4>Location</h4>
```

Comment:
Considering this a live event and attendees should be at it then might not need to give location and date!